pycm/pycm_util.py

```diff
@@ -727,8 +727,8 @@ def sort_char_num(input_list):
     :type input_list: iterable
     :return: a sorted list of strings
     """
-    sort_by = lambda x: [(x, False, False) if not re.findall(r'\d+', x)
-                         else (x[:re.search(r'\d+', x).start()],
-                               int(re.findall(r'\d+', x)[0]),
-                               x[re.search(r'\d+', x).end():])]
+    def sort_by(x): return [(x, False, False) if not re.findall(r'\d+', x)
+                            else (x[:re.search(r'\d+', x).start()],
+                                  int(re.findall(r'\d+', x)[0]),
+                                  x[re.search(r'\d+', x).end():])]
```

Comment:
umm, not sure about this.
Maybe adding it as another function (parallel to `sort_char_num`) would be a better choice - rather than in the `sort_char_num`.

Reply:
It's autopep8 script output.
But you are right, I will solve it ASAP 🚀

Reply:
Fixed in https://github.com/sepandhaghighi/pycm/pull/485/commits/625de03996fcdb19786c66d7a959c9b302ddcf05